imports/plugins/core/i18n/client/components/localizationSettings.js

```diff
@@ -0,0 +1,182 @@
+        <ListItem
+          actionType={"switch"}
+          key={key}
+          label={currency.label}
+          switchOn={currency.enabled}
+          switchName={currency.name}
+          onSwitchChange={this.props.onUpdateCurrencyConfiguration}
+        />
+      );
+    });
+  }
+
+  renderLanguages() {
+    return this.props.languages.map((language, key) => {
+      return (
+        <ListItem
+          actionType={"switch"}
+          key={key}
+          label={language.label}
+          switchOn={language.enabled}
+          switchName={language.value}
+          onSwitchChange={this.props.onUpdateLanguageConfiguration}
+        />
+      );
+    });
+  }
+
+  handleSubmit = (event, formData) => {
+    if (typeof this.props.onUpdateLocalization === "function") {
+      this.props.onUpdateLocalization(formData.doc);
+    }
+  }
+
+  handleAllOn = (event, name) => {
+    if (name === "language") {
+      if (typeof this.props.onEnableAllLanguages === "function") {
+        this.props.onEnableAllLanguages(true);
+      }
+    } else if (name === "currency") {
+      if (typeof this.props.onEnableAllCurrencies === "function") {
+        this.props.onEnableAllCurrencies(true);
+      }
+    }
+  }
+
+  handleAllOff = (event, name) => {
+    if (name === "language") {
+      if (typeof this.props.onEnableAllLanguages === "function") {
+        this.props.onEnableAllLanguages(false);
+      }
+    } else if (name === "currency") {
+      if (typeof this.props.onEnableAllCurrencies === "function") {
+        this.props.onEnableAllCurrencies(false);
+      }
+    }
+  }
+
+  renderListControls(name) {
+    return (
+      <CardToolbar>
+        <FlatButton
+          i18nKeyLabel={"all.allOn"}
+          label="All On"
+          value={name}
+          onClick={this.handleAllOn}
+        />
+        { "|" }
+        <FlatButton
+          i18nKeyLabel={"all.allOff"}
+          label="All Off"
+          value={name}
+          onClick={this.handleAllOff}
+        />
+      </CardToolbar>
+    );
+  }
+
+  render() {
+    return (
+      <CardGroup>
+        <SettingsCard
+          i18nKeyTitle="i18nSettings.localization"
+          name="localization"
+          packageName={PACKAGE_NAME}
+          saveOpenStateToPreferences={true}
+          showSwitch={false}
+          title="Shop Localization"
+        >
+          <Form
+            autoSave={true}
+            schema={ShopSchema}
+            doc={this.props.shop}
+            fields={{
+              timezone: {
+                type: "select",
+                options: this.props.timezoneOptions
+              },
+              currency: {
+                type: "select",
+                options: this.props.currencyOptions
+              },
+              baseUOM: {
+                type: "select",
+                options: this.props.uomOptions
+              },
+              language: {
+                type: "select",
+                options: this.props.currencyOptions
```

Comment:
Also when you try to select value(Base Language) from available options, expandable arrows from below 2 cards (Languages and Currency) are visible. See the  attached pic. 
![capture](https://cloud.githubusercontent.com/assets/14071646/23807817/3399526e-05ed-11e7-8e4f-272857fc78d0.PNG)